frontend/src/components/accountPage/UploadProgressModal/UploadProgressModal.tsx

```diff
@@ -0,0 +1,163 @@
+import { Button, Modal } from 'antd';
+import {
+  StopOutlined,
+  CaretRightOutlined,
+  CaretLeftOutlined,
+} from '@ant-design/icons';
+import { FC, useEffect, useState } from 'react';
+import { UserAccountPage } from '../../../utils';
+
+import UploadProgressContent from './UploadProgressContent';
+import { Role } from '../../../generated-types';
+import UploadProgressErrorsModal from './UploadProgressErrorsModal';
+export interface IUploadProgressModalInterface {
+  onClose: () => void;
+  confirmUpload: (
+    users: UserAccountPage[],
+    workspaces: any[]
+  ) => Promise<boolean>;
+  setAbortUploading: (value: boolean) => void;
+  setUploadingErrors: (errors: any) => void;
+  show: boolean;
+  workspaceName: string;
+  uploadedNumber: number;
+  abortUploading: boolean;
+  uploadingErrors: any[];
+  uploadedUserNumber: number;
+}
+
+const UploadProgressModal: FC<IUploadProgressModalInterface> = props => {
+  const { setAbortUploading } = props;
+  const [usersCSV, setUsersCSV] = useState<UserAccountPage[]>([]);
+  const [uploadingStatusResult, setUploadingStatusResult] = useState(false);
+  const [editing, setEditing] = useState<boolean>(false);
+  const [stepCurrent, setStepCurrent] = useState<number>(0);
+  const [stepStatus, setStepStatus] = useState<
+    'finish' | 'error' | 'process' | 'wait'
+  >('process');
+
+  const handleOk = async () => {
+    if (stepCurrent + 1 === 2) {
+      setStepStatus('wait');
+      setStepCurrent(2);
+
+      const result = await props.confirmUpload(usersCSV, [
+        { name: props.workspaceName, role: Role.User },
+      ]);
+      setUploadingStatusResult(result);
+      setStepCurrent(3);
+    } else {
+      setStepCurrent(number => number + 1);
```

Comment:
```suggestion
      setStepCurrent(stepCurrent + 1);
```
Makes me less anxsious